### ISSUE TITLE
Escaping Query Parameters

### ISSUE THREAD

**Issue by jhonyf** [ronaldwan/foursquare-api#2]
Under the foursquare_node.rb, you should escape the parameter values. URI.parse fails if it finds a space in the string it tries to parse.

**Comment by ronaldwan** [comment]
Right.  Thanks for the info.  I'm working on a new version to patch up several things.  Stay tuned!
